### PR TITLE
Fixed bug where git output was displayed after most commands.

### PR DESCRIPTION
**Pull request by laurencer**
\cc @SamRoberts @RayRoestenburg

### CODE REVIEW

**Comment by laurencer** [comment]
Note - it's not ideal but this means that if for some reason git is missing or otherwise broken, it will always think that the repository is a `SNAPSHOT` (and you won't see this in the logs). Does everyone think this is okay behaviour?

I could perhaps ensure that it only logs STDERR (but this would print out `fatal: Not a git repository (or any of the parent directories): .git`) when run in something that isn't a git repository...

\cc @stephanh

**Comment by SamRoberts** [comment]
Looks sensible. :+1: 

Does no other git command print output that we don't want?

**Comment by laurencer** [comment]
@SamRoberts - it didn't compile at all (e.g. SBT failed to start).

**Comment by laurencer** [comment]
Now it will output in red bold the standard error (where there is no git repository).

File: uniform-core/src/main/scala/au/com/cba/omnia/uniform/core/version/GitInfo.scala [hunk 1]
```diff
@@ -33,6 +33,15 @@ case class Clean(hash: String) extends GitStatus
 
 case class Dirty(hash: String) extends GitStatus
 
+/** Logs any STDERR output in bold red and discards STDOUT output */
+object OnlyLogStdErr extends ProcessLogger {
+  import _root_.scala.Console._
```
Comment (stephanh) [inline uniform-core/src/main/scala/au/com/cba/omnia/uniform/core/version/GitInfo.scala]: Can you move import to the top and use a qualified import?

File: uniform-core/src/main/scala/au/com/cba/omnia/uniform/core/version/GitInfo.scala [hunk 2]
```diff
@@ -33,6 +33,15 @@ case class Clean(hash: String) extends GitStatus
...
+
+  override def buffer[T](f: => T) = f
+  override def error(s: => String) = println(s"${RED}${BOLD}${s}${RESET}")
+  override def info(s: => String) = {}
```
Comment (stephanh) [inline uniform-core/src/main/scala/au/com/cba/omnia/uniform/core/version/GitInfo.scala]: Align `=` please.

**Comment by stephanh** [comment]
:+1:

**Pull request merged**